lib/os_cpuinfo.c

```diff
@@ -43,6 +43,7 @@
 #include <limits.h>
 #include <stdlib.h>
 #include <string.h>
+#include <sys/resource.h>
```

Comment:
Remove this inclusion. common.c has this file

Reply:
Done